### ISSUE TITLE
Navigation through outline applies unwanted zoom

### ISSUE THREAD

**Issue by Etiake** [VadimDez/ng2-pdf-viewer#1005]
##### Bug Report or Feature Request (mark with an `x`)
```
- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [ x] Bug report -> please search issues before submitting
- [ ] Feature request
- [ ] Documentation issue or request
```
Hello! 
Using your pdf viewer (last version 9.1.5) I noticed something strange with the use of outline. Clicking on a section of the outline, the pdf moves on the correct line, but the pdf zoom in or zoom out (based on the previous zoom of the pdf). You can check that from the demo: 

[https://vadimdez.github.io/ng2-pdf-viewer/](url)
 
 Is there a way to avoid this behaviour? Maybe with some configurations?
 
 Thank you for this amazing viewer ❤️

**Comment by Etiake** [comment]
Hoping to help, here a workaround (with typescript/angular, but it can be translated in other languages)

	navigateToRow(dest: string | unknown[]): void {
		const oldZoom = this.zoom;
		//reset zoom;
		this.zoom = 0;
		this.pdfComponent.pdfLinkService.goToDestination(dest).then(() => {
			_.delay(() => { 
				//return to old zoom after few milliseconds
				this.zoom = oldZoom;
			}, 10);
		});
	}
	
In other words, This code changes the value of the zoom for few milliseconds, then returns to the old zoom. It's a dirty way to solve this, but it works.

**Comment by Etiake** [comment]
Anyone having this issue?

**Comment by Etiake** [comment]
Anyone with this issue?

**Comment by Helias** [comment]
maybe this PR could solve this issue? https://github.com/VadimDez/ng2-pdf-viewer/pull/1199
I changed a bit the standard zooming behavior
